colorama/ansitowin32.py

```diff
@@ -66,13 +80,16 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # to support the ANSI codes.
         conversion_supported = on_windows and winapi_test()
 
+        # Does this version of Win 10 support ANSI?
+        win_10_ansi_support = on_windows and check_win_10_ansi_support()
+
         # should we strip ANSI sequences from our output?
-        if strip is None:
+        if strip is None and not win_10_ansi_support:
```

Comment:
I prefer to add the win10 check inside the if, this way "strip" will always be True/False and not be left as None. Same for 'convert'